src/common/transformers/BOOLEAN.js

```diff
@@ -0,0 +1,40 @@
+export const toBoolean = (value) => {
+    if (!value) {
+        return false;
+    }
+
+    if (Array.isArray(value)) {
+        return value.map(toBoolean);
+    }
+
+    if (typeof value === 'string') {
+        const val = value.trim();
+        if (val === '1' || value === 'true' || value === 'on') {
+            return true;
+        }
+        return false;
+    }
+
+    if (value === 1 || value === true) {
```

Comment:
This should almost never happen (except if casted just before, the input is a string).